files/zh-cn/learn/css/building_blocks/cascade_and_inheritance/index.md

```diff
@@ -165,9 +165,9 @@ CSS 的简写属性 [`all`](/zh-CN/docs/Web/CSS/all) 可以用于同时将这些
 
 一个选择器的优先级可以说是由三个不同的值（或分量）相加，可以认为是百（ID）十（类）个（元素）——三位数的三个位数：
 
-2. **ID**：选择器中包含 ID 选择器则百位得一分。
-3. **类**：选择器中包含类选择器、属性选择器或者伪类则十位得一分。
-4. **元素**：选择器中包含元素、伪元素选择器则个位得一分。
+- **Identifiers**：选择器中包含 ID 选择器则百位得一分。
```

Comment:
```suggestion
- **ID**：选择器中包含 ID 选择器则百位得一分。
```

Reply:
I can change it back, but they have it fully spelled out in the English version, which is why I changed it

Reply:
In zh-CN, we are used to use `ID` for element Identifiers. It's ok to change it back 😊

See also: line 166